api/src/main/java/io/strimzi/api/kafka/model/KafkaClusterSpec.java

```diff
@@ -84,9 +84,9 @@ public void setVersion(String version) {
         this.version = version;
     }
 
-    @Description("The KRaft metadata.version that should be used by this Kafka cluster. " +
-            "This field will be ignored when running in ZooKeeper mode. " +
-            "Defaults to {DefaultKafkaMetadataVersion}.")
+    @Description("The KRaft `metadata.version` used by the Kafka cluster. " +
+            "This property is ignored when running in ZooKeeper mode. " +
+            "If the property is not set, it defaults to {DefaultKafkaMetadataVersion} based on the `version` property.")
```

Comment:
Can we avoid using the Asciidoc variables such as `{DefaultKafkaMetadataVersion}` in the descriptions? The do not render in the CRDs and it looks weird / buggy in various interfaces parsing the CRDs.

Reply:
I've taken them all out